task/src/main/java/no/nav/vedtak/felles/prosesstask/impl/DatabaseTypeTjeneste.java

```diff
@@ -0,0 +1,39 @@
+package no.nav.vedtak.felles.prosesstask.impl;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import no.nav.vedtak.konfig.KonfigVerdi;
+
+@ApplicationScoped
+public class DatabaseTypeTjeneste {
+
+    private static final Logger logger = LoggerFactory.getLogger(DatabaseTypeTjeneste.class);
+
+    private boolean erOracle;
+    private boolean erPostgres;
+
+    DatabaseTypeTjeneste() {
+        //for CDI proxy
+    }
+
+    @Inject
+    public DatabaseTypeTjeneste(@KonfigVerdi(value = "defaultDS.url", required = false) String ds) {
+        erOracle = ds != null && ds.startsWith("jdbc:oracle:");
```

Comment:
Sjekk datasource, evt dialectd (se under) i stedt for konfig.   Dette fordi det er flere måter å konfigugrere datasource på enn jdbc url.  Datasource vil nok være best.

Eksempel sjekk på dialect : https://github.com/navikt/fp-prosesstask/blob/0aa4bb6d6047a048d156e422e9a4ccb9884b80f6/task/src/main/java/no/nav/vedtak/felles/prosesstask/impl/ProsessTaskRepositoryImpl.java#L375)

Reply:
skal være noe kode i felles såvidt jeg husker som gjør noe sjekk på oracle via datasource. hør evt. med Mayu